src/hdf5.cpp

```diff
@@ -794,6 +793,11 @@ void readSolverSettingsFromHDF5(H5::H5File const& file, Solver &solver,
                     getDoubleScalarAttribute(file, datasetPath, "quad_rtol"));
     }
 
+    if(attributeExists(file, datasetPath, "ss_tol_factor")) {
```

Comment:
Could you please also update `writeSolverSettingsToHDF5` (same file) accordingly?

Reply:
:see_no_evil: